tests/conftest.py

```diff
@@ -63,6 +70,27 @@ def log_has_re(line, logs):
                   False)
 
 
+def num_log_contains(line, logs, count=False):
+    # caplog mocker returns log as a tuple: ('freqtrade.something', logging.WARNING, 'spamfoobar')
+    # and we want to check how many times line ('foo', for example) is contained in the tuples.
+    return sum(line in logger_name or line in message
+               for logger_name, level, message in logs.record_tuples)
+
+
+def num_log_has(line, logs, count=False):
+    # caplog mocker returns log as a tuple: ('freqtrade.something', logging.WARNING, 'foobar')
+    # and we want to check how many times line is presente in the tuples.
+    return sum(line == logger_name or line == message
+               for logger_name, level, message in logs.record_tuples)
+
+
+def num_log_has_re(line, logs, count=False):
```

Comment:
why do all of these methods have an unused `count` argument?

Reply:
It was a leftover from previous experiments with the counting methods. Removed now.